lib/figma.js

```diff
@@ -0,0 +1,83 @@
+const Figma = require('figma-js');
+
+let elements = [];
+
+/**
+ * Helper function for flattenChildren
+ * @param {Object} element a parent element
+ * @param {Arrat<Object>} resultArray the array that will be returned at the end
+ */
+const flattenChildrenHelper = (element, resultArray) => {
+  if (element.children) {
+    element.children.forEach(child => {
+      const clone = Object.assign({}, child);
+      resultArray.push(clone);
+      flattenChildrenHelper(child, resultArray);
+    });
+  }
+};
+
+/**
+ * Flattens a nested Figma JSON file recursively using
+ * flattenChildrenHelper.
+ * @param {Object} element parent element of the JSON file
+ * @return {Array<Object>}
+ */
+const flattenChildren = element => {
+  const flattened = [];
+  const clone = Object.assign({}, element);
+  flattened.push(clone);
+  element.children.forEach(child => {
+    flattenChildrenHelper(child, flattened);
+  });
+  return flattened;
+};
+
+/**
+ * Transforms a Figma formatted JSON object
+ * to a standardized JSON object
+ * and initializes the local variable 'elements'
+ * @param {Object} file Figma formatted JSON file
+ * @return {void}
+ */
+const figma2jsonFromJSON = json => {
```

Comment:
This is a bit weird. So this is just the mapping to get Figma API -> Tokens? If so, just name it `figmaToTokens`